Datastore/src/Operation.php

```diff
@@ -395,11 +461,27 @@ public function lookup(array $keys, array $options = [])
             $serviceKeys[] = $key->keyObject();
         });
 
-        $res = $this->connection->lookup($options + $this->readOptions($options) + [
+        list($data, $optionalArgs) = $this->splitOptionalArgs($options, [
```

Comment:
I don't see these keys being used in the `LookupRequest` so naturally they should go into `$optionalArgs` but I also don't see them in the `DatastoreClient::lookup` method's doc.

Reply:
These keys have already been consumed by the upstream layers and are redundant going forward to transport layer.

It makes sense to just pluck them out of array. Please see https://github.com/yash30201/google-cloud-php/pull/4